lms/db/_bulk_action.py

```diff
@@ -50,6 +51,9 @@ class Config:
         upsert_update_elements: List[str]
         """Columns to update when a match is found."""
 
+        upsert_update_datetime_column: Optional[str] = None
```

Comment:
Instead of using the config we could do isinstance(model_class, CreateUpdateMixin) which is probably what you want most of the time but it's a bit more magical.